plugins/c8y_firmware_plugin/src/main.rs

```diff
@@ -10,15 +10,12 @@ use crate::download::DownloadManager;
 use crate::download::DownloadRequest;
 use crate::download::DownloadResponse;
 use crate::error::FirmwareManagementError;
+use crate::firmware_manager::create_directories;
```

Comment:
This `create_directories` is called only in this file. Why did you define it in a different file and import it?

Reply:
Just for future refactoring in the actor model where we won't be using this `main` anymore. So, thought of keeping all the "portable" code in the `firmware_manager` itself.